rpcserver.go

```diff
@@ -790,19 +811,11 @@ func (r *rpcServer) ListUnspent(ctx context.Context,
 			return nil, fmt.Errorf("invalid utxo address type")
 		}
 
-		// Now that we know we have a proper mapping to an address,
-		// we'll convert the regular outpoint to an lnrpc variant.
-		outpoint := &lnrpc.OutPoint{
-			TxidBytes:   utxo.OutPoint.Hash[:],
-			TxidStr:     utxo.OutPoint.Hash.String(),
-			OutputIndex: utxo.OutPoint.Index,
-		}
-
 		utxoResp := lnrpc.Utxo{
 			Type:          addrType,
 			AmountSat:     int64(utxo.Value),
 			PkScript:      hex.EncodeToString(utxo.PkScript),
-			Outpoint:      outpoint,
+			Outpoint:      marshallOutPoint(&utxo.OutPoint),
```

Comment:
Alternatively, you can just pass the outpoint as is, and then within `lncli` use `chainhash.NewFromStr` (or w/e it's called). This'll result in the bytes only being reversed when they need to be displayed to the user.

Reply:
We can't use the string if the response doesn't include it, which we decided in a previous comment? For what it's worth, I've also seen some confusion with regular users of the RPC expecting the bytes to not be in the reversed order.

Reply:
The response has the bytes, why do you need the string? This is about _where_ do we reverse the bytes. What txids aren't displayed in reversed order today? Perhaps they get confused if something is in base58, then they need to decode _and_ reverse it manually.

Reply:
Ah I see what you mean about the string, even then it can just be reversed on the cli, we don't reverse anywhere else atm.

Reply:
Fixed.